### DISCUSSION TITLE
SECTION 1 LESSON 6 - ANY IDEA HOW HE GETS THE ADDRESS FROM METAMASK

### DISCUSSION THREAD

**Discussion by archivedtim** [Cyfrin/foundry-full-course-cu#2694] [Q&A]
currently stuck here

**Comment by EngrPips** [comment]
Hello you can click on the part i put in red box in the image below to copy your address.

<img width="360" alt="Screenshot 2024-10-17 at 08 35 40" src="https://github.com/user-attachments/assets/192eccaa-356b-4a23-8048-209626841dca">
